lib/b2b/index.ts

```diff
@@ -81,8 +80,8 @@ export type {
 
 export type {
   B2BOrganizationsMembersOAuthProvidersGoogleResponse,
+  B2BOrganizationsMembersOAuthProvidersMicrosoftRequest,
   B2BOrganizationsMembersOAuthProvidersMicrosoftResponse,
-  B2BOrganizationsMembersOAuthProvidersProviderInformationRequest,
```

Comment:
I think you need to update either protobuf or sdk-codegen

Reply:
And then probably reset _this_ repo to re-add the manually added type (before regenerating)

Reply:
Was just going through this! Should be good now. Thx Logan!